src/components/Button/index.tsx

```diff
@@ -294,6 +307,17 @@ export function ButtonWithLink({ link, text, style }: { link: string; text: stri
   )
 }
 
+export function ButtonExternalLink({ link, children }: { link: string; children: any }) {
+  return (
+    <ButtonPurpleDim as={ExternalLink} href={link}>
+      {children}
+      <Box ml={2}>
+        <ArrowUpRight size="14px" color={colors(true).purple2} />
```

Comment:
What do you mean by color provider? I didn't know we had one, so I just imported the colors and passed true because the function needs a darkmode (boolean) argument.